### ISSUE TITLE
Vitune playlist sync problem.

### ISSUE THREAD

**Issue by Nikichelak** [25huizengek1/ViTune#517]
### Steps to reproduce the bug

1. Firstly, sync a playlist from piped to Vitune... (That is, registering and logging in the piped...)
2. Go to playlist section in vitune
3. Go to the synced playlist..

### Expected behavior

The playlist keeps on buffering(loading)... But, it never shows the music of the playlist...
![Screenshot_20241025_205321](https://github.com/user-attachments/assets/57c5b120-a0ad-4515-bd63-e3c87bee4968)

### Actual behavior

The playlist should show result in few seconds..

### Screenshots/Screen recordings

![Screenshot_20241025_205321](https://github.com/user-attachments/assets/72f7cd5d-80a4-4cac-a197-c4386eda80fe)

### ViTune version

v1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 14

### Device info

I am using A "A9+ samsung tablet"...

### Checklist

- [X] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by 25huizengek1** [comment]
This issue is not related to ViTune. All Piped instances are getting blocked by YouTube and can't get any video metadata, so Piped doesn't return any playlists right now.

**Comment by Nikichelak** [comment]
All right... Thanks for your help and reply... I'm very grateful...
